R/apply_theme.R

```diff
@@ -443,9 +457,15 @@ apply_theme <- function(gg, visR_theme_dict = NULL) {
   # Reset background
   gg <- gg + ggplot2::theme_minimal()
 
+  if (!skipcolor) {
+    gg <- gg +
+      ggplot2::scale_colour_manual(labels = names(cols),
+                                   values = cols,
+                                   aesthetics = c("colour", "fill"), na.value = "grey50") +
+      guides(color=guide_legend(ttl))
```

Comment:
add ggplot2 to guides on line 465 

`ggplot2::guides(color=guide_legend(ttl))`